Makefile

```diff
@@ -36,6 +36,10 @@ test:
 	# run wasmedge test without the default `static` feature
 	RUST_LOG=trace cargo test --package containerd-shim-wasmedge --verbose --no-default-features --features standalone -- --nocapture
 
+.PHONY: setup-env
+setup-env:
+	sudo ./scripts/setup.sh
```

Comment:
I remember that @cpuguy83 had this comment https://github.com/containerd/runwasi/pull/186#pullrequestreview-1538019005.

Reply:
Sounds good, I don't have a strong opinion here other than we could provide a script for the readme so the deps stay in sync.

Reply:
switched to using a single script and no make file